code/modules/reagents/chemistry/reagents/alcohol.dm

```diff
@@ -536,7 +535,7 @@
 
 /datum/reagent/consumable/ethanol/beepsky_smash/on_mob_life(mob/living/M)
 	var/update_flag = STATUS_UPDATE_NONE
-	update_flag |= M.Stun(1, FALSE)
+	M.Stun(2 SECONDS)
```

Comment:
Are there any implications to not setting the `update_flag`? Both here and other reagent `on_mob_life`s. Admittedly I don't know how important it is.

Reply:
The update flag was used to update the statuses incased they got applied/removed. This is all handled in the status effect.